WordPress/Classes/Services/BlogService.m

```diff
@@ -640,6 +640,7 @@ - (OptionsHandler)optionsHandlerWithBlogObjectID:(NSManagedObjectID *)blogObject
                                                                            error:nil];
             if (blog) {
                 blog.options = [NSDictionary dictionaryWithDictionary:options];
+                blog.isAdmin = YES;
```

Comment:
Shouldn't we be parsing this from somewhere? (Otherwise everyone will be an admin, unless i'm missing a huge something!)